src/cromshell/utilities/IOUtils.py

```diff
@@ -0,0 +1,39 @@
+from termcolor import colored
+from pathlib import Path
+
+
+def copy_file(source, destination):
+    source_path = Path(source)
+    destination_path = Path(destination)
+    destination_path.write_bytes(source_path.read_bytes())
+
+
+def turtleDead(quiet):
+    if not quiet:
+        print(colored("    ,,    ,,     ,,      ", 'red'))
+        print(colored("    \\‾\\,-/‾/====/‾/,   ", 'red'))
+        print(colored("     \\/=<‾><‾><‾><‾>-,  ", 'red'))
+        print(colored("     / (\\‾\\‾|‾/‾/‾/‾   ", 'red'))
+        print(colored("    \\‾x/ ˙'-;-;-'˙      ", 'red'))
+        print(colored("     ‾‾                  ", 'red'))
+
+def turtle(quiet):
```

Comment:
do you prefer to have two lines between functions, or just one? either way let's make it consistent.
And while trivial, they are better with docstrings

Reply:
done, two lines